modules/oadp-1-4-0-release-notes.adoc

```diff
@@ -0,0 +1,71 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-06-28
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-1-4-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="oadp-1-4-0-release-notes_{context}"]
+= OADP 1.4.0 release notes
+
+The {oadp-first} 1.4.0 release notes lists resolved issues and known issues.
+
+[id="resolved-issues-1-4-0_{context}"]
+== Resolved issues
+
+.Restore works correctly in {product-title} 4.16
```

Comment:
I understand this his how OADP release notes were previously formatted, but I strongly urge you to reconsider adding a "title" to the bugs for a couple reasons: 
1. It's hard to read, as the text is so small, in red, against a white background, in italics. 
2. OpenShift docs do not do this: [Known Issues](https://docs.openshift.com/container-platform/4.16/release_notes/ocp-4-16-release-notes.html#ocp-4-16-known-issues_release-notes)
3. A new assembly is a good time to make good housekeeping changes going forward, and I think matching release notes style is an important template to follow.

Reply:
I'll share this suggestion with the team. For now, keeping the same flow for consistency.